int32[8] right_arm
int32[8] left_arm
int32 vel
bool speed
